.github/workflows/static.yml

```diff
@@ -9,17 +9,16 @@ on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
-# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+# Sets the GITHUB_TOKEN permissions to allow deployment to GitHub Pages
```

Comment:
line 5 is wrong: can you please update it for main branch! 

our ci/cd process will be as follows, can you redesign please your CI/CD according to it. 

`-> build -> run the tests -> PR(code-review) -> approval -> merge into master -> deploy to github pages`